abc.templates/deployments/spec.yaml

```diff
@@ -21,23 +21,55 @@ inputs:
   - name: 'subdirectory'
     desc: 'The subdirectory to render the template'
     default: 'deployments/github-token-minter'
+  - name: 'wif_provider'
+    desc: 'The Google Cloud workload identity federation provider for GitHub Token Minter'
+  - name: 'wif_service_account'
+    desc: 'The Google Cloud service account for GitHub Token Minter'
+    rules:
+      - rule: 'gcp_matches_service_account(wif_service_account)'
+  - name: 'project_id'
+    desc: 'The Google Cloud project ID containing GitHub Token Minter'
+    rules:
+      - rule: 'gcp_matches_project_id(project_id)'
+  - name: 'full_image_name'
+    desc: 'The full image name, excluding the version tag'
   - name: 'version'
     desc: 'The GitHub Token Minter version tag'
+  - name: 'region'
+    desc: 'The region of the GitHub Token Minter Cloud Run service'
+  - name: 'service_name'
+    desc: 'The name of the GitHub Token Minter Cloud Run service'
 
 steps:
   - desc: 'Include required Dockerfiles and configs'
     action: 'include'
     params:
       paths:
         - 'contents'
+        - 'contents/workflows'
```

Comment:
Should these be siblings instead? It seems strange that contents includes all children but then workflows is "de-referenced".